Makefile

```diff
@@ -608,28 +610,13 @@ ifeq ($(OPERATING_SYSTEM),Darwin)
 		"./build/libs/$(OPERATING_SYSTEM)/$(PROCESSOR)/$(TARGET)/_deps/hacl-build/libhacl.dylib" 
 endif
 
-test-netstandard-copy-output-ui:
```

Comment:
we shouldnt need to copy the output like this anymore. i left the one for the tests because i havent had a chance to test it yet.